code/modules/mining/mine_turfs.dm

```diff
@@ -303,6 +303,7 @@ turf/unsimulated/mineral/ChangeTurf(var/turf/N, var/tell_universe=1, var/force_l
 						B = getFromPool(/obj/structure/boulder, src)
 						if(artifact_find)
 							B.artifact_find = artifact_find
+							B.investigation_log(I_ARTIFACT, "|| [B.artifact_find.artifact_find_type] || found by [user.name]([user.ckey]).")
```

Comment:
It is using user from `/turf/unsimulated/mineral/attackby(obj/item/weapon/W as obj, mob/user as mob)`
![user](https://cloud.githubusercontent.com/assets/17928298/21966267/5014fc54-db4f-11e6-83ad-34ec96173614.png)
This log is from before adding that second ||